src/gui/dialogs/addon/addon_auth.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+	Part of the Battle for Wesnoth Project https://www.wesnoth.org/
+
+	This program is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; either version 2 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY.
+
+	See the COPYING file for more details.
+*/
+
+#pragma once
+
+#include "gui/dialogs/modal_dialog.hpp"
+
+namespace gui2
+{
+class field_text;
```

Comment:
Unnecessary forward definition, please remove and collapse the namespaces.

Reply:
Fixed.